common/src/main/java/org/broadleafcommerce/common/resource/service/ResourceBundlingServiceImpl.java

```diff
@@ -448,4 +450,30 @@ protected Charset getBundleCharSet() {
             return Charset.forName(charsetProperty);
         }
     }
+
+    @Override
+    public List<String> findBundlesNameByResourceFileName(String fileName){
+        List<String> result = new ArrayList<>();
+        for (Entry<String, BundledResourceInfo> bundleInfo : createdBundles.entrySet()) {
+            BundledResourceInfo value = bundleInfo.getValue();
+            for (String s : value.getBundledFilePaths()) {
+                String f = FilenameUtils.removeExtension(fileName);
+                if(s.startsWith(f)){
```

Comment:
Would there be any possible conflict due to using `startsWith`?

I could imagine there being two files like `nav.css` and `nav-mobile.css`, in which case a change to `nav.css` would also find the bundle for `nav-mobile.css`. Although it's not likely to be a problem right now, since this is only used for invalidating the bundles, and files like this would likely exist in the same bundle anyway.

Reply:
@epenning thx for pointing that out but I think I did that for some good reason, I don't remember exactly but I believe that there was some suffix of some hash, so I can't compare names exactly

Reply:
@epenning I checked it to see why this check did. That make sense, because to file path hash added. For example it's looks like : nav-css-tf167635363.css